packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/flex.dart

```diff
@@ -70,78 +74,179 @@ const crossAxisLightColor = Color(0xFFB3D25A);
 const crossAxisDarkColor = Color(0xFFB3D25A);
 const crossAxisColor = ThemedColor(crossAxisLightColor, crossAxisDarkColor);
 
-const mainAxisLightTextColor = Color(0xFF913549);
-const mainAxisDarkTextColor = Color(0xFFEA637C);
+const mainAxisTextColorLight = Color(0xFF913549);
+const mainAxisTextColorDark = Color(0xFFEA637C);
 const mainAxisTextColor =
-    ThemedColor(mainAxisLightTextColor, mainAxisDarkTextColor);
+    ThemedColor(mainAxisTextColorLight, mainAxisTextColorDark);
 
-const crossAxisLightTextColor = Color(0xFF66672C);
-const crossAxisDarkTextColor = Color(0xFFB3D25A);
+const crossAxisTextColorLight = Color(0xFF66672C);
+const crossAxisTextColorsDark = Color(0xFFB3D25A);
 const crossAxisTextColor =
-    ThemedColor(crossAxisLightTextColor, crossAxisDarkTextColor);
+    ThemedColor(crossAxisTextColorLight, crossAxisTextColorsDark);
+
+const overflowBackgroundColorDark = Color(0xFFB00020);
+const overflowBackgroundColorLight = Color(0xFFB00020);
+const overflowBackgroundColor =
+    ThemedColor(overflowBackgroundColorLight, overflowBackgroundColorDark);
+
+const overflowTextColorDark = Color(0xFFFFFFFF);
+const overflowTextColorLight = Color(0xFFFFFFFF);
+const overflowTextColor =
+    ThemedColor(overflowTextColorLight, overflowTextColorDark);
 
 const freeSpaceAssetName = 'assets/img/story_of_layout/empty_space.png';
 
 const entranceAnimationDuration = Duration(milliseconds: 500);
 
+const defaultDimensionIndicatorTextStyle = TextStyle(
+  height: 1.0,
+);
+
+Widget _dimensionDescription(TextSpan description, bool overflow) {
+  final text = Text.rich(
+    description,
+    textAlign: TextAlign.center,
+    style: TextStyle(
```

Comment:
define a couple TextStyle constants one for overflow and one for regular.
There are conditionals in a few different places here so might as well.